client/src/components/TimetableSwitcher/TimetableSwitcher.tsx

```diff
@@ -58,6 +64,25 @@ const TimetableSwitcher = memo(() => {
 		else setAction(inputAction);
 	};
 
+    const handleSaveTimetable = async () => {
+        const input = document.getElementById('timetable') as HTMLElement;
+        const imgData = await domtoimage.toPng(input)
+        triggerBase64Download(imgData, 'FFCSThingy - Timetable');
+    };
+
+    const handleSaveCourses = async () => {
+        let csv = '';
+        let header = Object.keys(timetable[0]).filter((val, index) => index!==0).join(',');
+        let values = timetable.map(o => Object.values(o).filter((val, index) => index!==0).join(',')).join('\n');
+        csv += header + '\n' + values;
```

Comment:
Try to use `const` here instead.
```suggestion
        const header = Object.keys(timetable[0]).filter((val, index) => index!==0).join(',');
        const values = timetable.map(o => Object.values(o).filter((val, index) => index!==0).join(',')).join('\n');
        const csv = `${header}\n${values}`;
```